components/playlist/browser/DEPS

```diff
@@ -9,5 +9,6 @@ include_rules = [
   "+services/network/public/cpp",
   "+services/preferences/public/cpp",
   "+third_party/blink/public/common/web_preferences",
+  "+third_party/blink/public/common/user_agent",
```

Comment:
just make it `"+third_party/blink/public/common"`, because all these subdirs are a part of `common` target.